src/Router/Matchers.ts

```diff
@@ -30,8 +30,11 @@ export class RouteMatchers extends Macroable implements RouteMatchersContract {
   /**
    * Enforce value to be formatted as uuid
    */
-  public uuid(): { match: RegExp } {
-    return { match: /^[0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{12}$/ }
+  public uuid(): { match: RegExp; cast: (value: string) => string } {
+    return {
+      match: /^[0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{12}$/,
+      cast: (value) => value.toLowerCase(),
```

Comment:
Are-you sure? You meas toUpperCase or you must update the regex to use lower case instead of upper case.

Reply:
or you can use a regex to match both : 

```js
const regex = /^[0-9a-zA-F]{8}-[0-9a-zA-F]{4}-[0-9a-zA-F]{4}-[0-9a-zA-F]{4}-[0-9a-zA-F]{12}$/
```

Reply:
Yup. I think you are still only allowing uppercase values. So regex must be updated

Reply:
Somehow missed it. Today I will commit the corrected regex.

Reply:
I think now you can do the review again